### PR TITLE
WFNEWS-1042 Evac Alert Full Details

### PR DESCRIPTION
**Pull request by ssylver93**

### CODE REVIEW

File: client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  <div class="info"><img height="20" width="20" class="icon" src="/assets/images/svg-icons/location-pin.svg"
+      alt="Location Pin" />Issued by {{evacData.issuingAgency}}</div>
+  <div class="info"><img height="20" width="20" class="icon" src="/assets/images/svg-icons/carbon_layers.svg"
+      alt="Layers" />Fire Evacuation Orders and Alerts Layer</div>
+</div>
+<hr>
+<div class="info-container" *ngIf="incident">
+  <div class="full-details-secondary-title">Associated Wildfire</div>
+  <div class="fire-info-container">
+    <div class="wildfire-title">{{incident.incidentName}}<img height="20" width="20" class="wildfire-title-icon"
```
Comment (yzlucas) [inline client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.html:23]: might want to clean up these 20, 20. And I see there is no wildfire-title-icon class in the files

File: client/wfnews-war/src/main/angular/src/app/components/full-details/evac-other-info/evac-other-info.component.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  templateUrl: './evac-other-info.component.html',
+  styleUrls: ['./evac-other-info.component.scss']
+})
+export class EvacOtherInfoComponent {
+
+  constructor(private appConfigService: AppConfigService) {
+
+  }
+
+  wildfirePreparedness() {
```
Comment (yzlucas) [inline client/wfnews-war/src/main/angular/src/app/components/full-details/evac-other-info/evac-other-info.component.ts:15]: Not important, but I think we can simplify these window.open() to one method 
ex: 
openLink(link: string) {
window.open(this.appConfigService.getConfig().externalAppConfig[link] as unknown as string, '_blank')
}
and just pass different variable with those (click) call in html file.

**Review by yzlucas — APPROVED** [review]

File: client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.html (hunk 1 above)
Comment (ssylver93) [reply client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.html]: Thanks I fixed them now

File: client/wfnews-war/src/main/angular/src/app/components/full-details/evac-other-info/evac-other-info.component.ts (hunk 2 above)
Comment (ssylver93) [reply client/wfnews-war/src/main/angular/src/app/components/full-details/evac-other-info/evac-other-info.component.ts:15]: Good idea!

**Pull request merged**